GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -2729,6 +2730,15 @@ private void Amend_CheckedChanged(object sender, EventArgs e)
             {
                 Message.Text = Module.GetPreviousCommitMessages(1).FirstOrDefault()?.Trim();
             }
+            UpdateCommitAndPushButton();
+        }
+
+        private void UpdateCommitAndPushButton()
+        {
+            if (AppSettings.CommitAndPushForcedWhenAmend)
+            {
+                CommitAndPush.BackColor = Amend.Checked ? Color.Salmon : SystemColors.ButtonFace;
```

Comment:
we need to ensure this will look good on Windows with dark/custom theme.
Maybe it will need to be configurable (can be in a separate PR in that case)

Reply:
Now it looks like that:
![gitext](https://user-images.githubusercontent.com/1003909/34512886-64379f4e-f065-11e7-881d-857dce54969c.png)
Is it good?

Reply:
I suppose it is ok, commit looks horrible though, heh.